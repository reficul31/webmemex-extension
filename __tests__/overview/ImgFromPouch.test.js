import React from 'react'
import ImgFromPouch from 'src/overview/components/ImgFromPouch'
import renderer from 'react-test-renderer'

describe('ImgFromPouch snapshot tests', () => {
	test('should test against the given snapshot', () => {
		const component = renderer.create(
			<ImgFromPouch />
			)
		let tree = component.toJSON()
		expect(tree).toMatchSnapshot()
	})
})