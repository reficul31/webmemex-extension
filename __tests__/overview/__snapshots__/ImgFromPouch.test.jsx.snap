// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImgFromPouch snapshot tests should test against the given snapshot 1`] = `
<img
  src={undefined}
/>
`;
